android/src/main/java/com/swmansion/gesturehandler/core/GestureHandler.kt

```diff
@@ -788,6 +803,10 @@ open class GestureHandler<ConcreteGestureHandlerT : GestureHandler<ConcreteGestu
   val lastPositionInWindowY: Float
     get() = lastAbsolutePositionY + lastEventOffsetY - windowOffset[1]
 
+  fun getTrackedPointersIDs() = trackedPointerIDs
+  fun getCachedPointersIDs() = cachedPointersIDs
+  fun getNextCachedPointerID() = nextCachedPointerID
```

Comment:
Why not just make them public? They're `vals` anyway so they cannot be set and the getter doesn't protect you from modifying the collection.

(getNextCachedPointerID may not be necessary at all)

Reply:
Done in [d0081f3](https://github.com/software-mansion/react-native-gesture-handler/pull/2759/commits/d0081f372715604f391c7919fea49e811cc8f3a5)